bin/s2nd.c

```diff
@@ -189,6 +190,8 @@ void usage()
                     "    Ex: --psk psk_id,psk_secret,SHA256 --psk shared_id,shared_secret,SHA384.\n");
     fprintf(stderr, "  -E, --max-early-data \n");
     fprintf(stderr, "    Sets maximum early data allowed in session tickets. \n");
+    fprintf(stderr, "  -N --npn \n");
+    fprintf(stderr, "    Indicates support for the NPN extension. The alpn option MUST be used with this option to signal the protocols supported."); 
```

Comment:
Is the "alpn" option documented in this usage method? Maybe I'm missing it, but I don't see it anywhere.